crates/optimism/node/Cargo.toml

```diff
@@ -13,25 +13,26 @@ workspace = true
 [dependencies]
 # reth
 reth-chainspec.workspace = true
-reth-primitives-traits.workspace = true
-reth-payload-builder.workspace = true
 reth-consensus.workspace = true
+reth-engine-local.workspace = true
+reth-evm.workspace = true
+reth-network.workspace = true
 reth-node-api.workspace = true
 reth-node-builder.workspace = true
-reth-tracing.workspace = true
+reth-node-core.workspace = true
+reth-payload-builder.workspace = true
+reth-primitives-traits.workspace = true
 reth-provider.workspace = true
-reth-transaction-pool.workspace = true
-reth-network.workspace = true
-reth-evm.workspace = true
-reth-trie-db.workspace = true
-reth-rpc-server-types.workspace = true
+reth-rpc-api.workspace = true
+reth-rpc-engine-api.workspace = true
 reth-rpc-eth-api.workspace = true
 reth-rpc-eth-types.workspace = true
+reth-rpc-server-types.workspace = true
 reth-tasks = { workspace = true, optional = true }
+reth-tracing.workspace = true
+reth-transaction-pool.workspace = true
 reth-trie-common.workspace = true
-reth-node-core.workspace = true
-reth-rpc-engine-api.workspace = true
-reth-rpc-api.workspace = true
+reth-trie-db.workspace = true
```

Comment:
please avoid reordering all of these in this pr, can be a separate pr, but doing this in a feature pr will only attract conflicts and additional review effort